cirq-core/cirq/protocols/json_test_data/DiagonalGate.repr

```diff
@@ -0,0 +1 @@
+cirq.DiagonalGate(diag_angles_radians=[0.0, 1.0, -1.0, 0.0])
```

Comment:
it's strange that the repr turns the values to a list instead of keeping it a tuple, but I see that's how it's always been and isn't from this PR.